metrics/gc.js

```diff
@@ -2,14 +2,17 @@
 const {PerformanceObserver, constants} = require('perf_hooks')
 
 const NODEJS_GC_DURATION_SECONDS = 'nodejs_gc_duration_seconds'
-const DEFAULT_GC_DURATION_BUCKETS = [0.001, 0.01, 0.1, 1, 2, 5]
+// const DEFAULT_GC_DURATION_BUCKETS = [0.001, 0.01, 0.1, 1, 2, 5]
 
+/**
+ * @param {import('@opentelemetry/api-metrics').Meter} meter 
+ * @param {*} config 
+ */
 module.exports = (meter, {prefix, labels, gcDurationBuckets}) => {
-  const boundaries = gcDurationBuckets || DEFAULT_GC_DURATION_BUCKETS
+  // const boundaries = gcDurationBuckets || DEFAULT_GC_DURATION_BUCKETS
 
   const histogram = meter.createHistogram(prefix + NODEJS_GC_DURATION_SECONDS, {
-    description: 'Garbage collection duration by kind, one of major, minor, incremental or weakcb.',
-    boundaries
+    description: 'Garbage collection duration by kind, one of major, minor, incremental or weakcb.'
   })
```

Comment:
@marcbachmann, I believe there's no way to pass it anymore

Reply:
The annotation is still outdated: https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-api-metrics/src/types/Meter.ts#L85

But it returns an ObservableGauge instance, which has register functions (Observable interface):
https://github.com/open-telemetry/opentelemetry-js/commit/ba3e3207edd217366710e10a769ba7ad4c4fa655#diff-c9a82ab13ebd74152ffa3cc91afa636f11f19174034c8bc12990ae2a69c2c507R11-R14

https://open-telemetry.github.io/opentelemetry-js/interfaces/_opentelemetry_api_metrics.Observable.html

Reply:
regarding boundaries, this is getting complicated 🙃 
https://github.com/open-telemetry/opentelemetry-js/issues/2999

It now belongs into a view configuration, so maybe we can just leave it out like you've done: https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-sdk-metrics-base/test/MeterProvider.test.ts#L181-L184